rnndb/graph/gf100_3d.xml

```diff
@@ -300,6 +300,10 @@ xsi:schemaLocation="http://nouveau.freedesktop.org/ rules-ng.xsd">
 		<reg32 offset="0x0a0c" name="VIEWPORT_TRANSLATE_X" length="16" stride="32" type="float"/>
 		<reg32 offset="0x0a10" name="VIEWPORT_TRANSLATE_Y" length="16" stride="32" type="float"/>
 		<reg32 offset="0x0a14" name="VIEWPORT_TRANSLATE_Z" length="16" stride="32" type="float"/>
+		<reg32 offset="0x0a1c" name="CONSERVATIVE_RASTER_SUBPIXEL_PRECISION_BIAS" variants="GM204_3D-">
```

Comment:
That's like 2x longer than all the other names. How about just calling it "SUBPIXEL_PRECISION" or something? That way it'll be SUBPIXEL_PRECISION = {BIAS_X = 5, ...}